frontend/src/metabase/query_builder/components/filters/pickers/RelativeDatePicker.jsx

```diff
@@ -83,7 +82,7 @@ export default class RelativeDatePicker extends Component {
             }
             intervals={intervals}
             formatter={formatter}
-            periods={this.props.hideTimeSelectors ? DATE_PERIODS : ALL_PERIODS}
+            periods={ALL_PERIODS}
```

Comment:
@paulrosenzweig , what about just doing it this way?

Reply:
Ah that's the better move. Removing it in `DateAllOptionsWidget.jsx` would allow times for the "on specific date" filter. I'm not sure if we want that, but my sense is "no".